Controllers/WebHookController.cs

```diff
@@ -0,0 +1,37 @@
+using JewishBot.WebHookHandlers.Telegram;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
+using Telegram.Bot;
+using Telegram.Bot.Types;
+using Telegram.Bot.Types.Enums;
+
+namespace JewishBot.Controllers
+{
+    public class WebHookController : Controller {
+        private readonly TelegramBotClient _bot;
```

Comment:
I would just leave out the underscore character in field names because it's already understood by looking at the first character of the word that it is a field. Maybe it's just my preference and, of course, you can completely ignore this suggestion :-)